Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -934,8 +935,10 @@ namespace FluentAssertions.Equivalency
     public interface IObjectInfo
     {
         System.Type CompileTimeType { get; }
+        System.Type ParentType { get; }
```

Comment:
I think we can apply the same reasoning for `IObjectInfo` as to `INode`, that this interface is only read and never re-implemented, so I'm also fine with this change.

Reply:
Yeah, I was pondering on this same issue, but settled my own doubts with the same assumption.